### PR TITLE
fix: skip DTG1 packets in parseSei

### PR DESCRIPTION
**Pull request by JulianDomingo**
Update of https://github.com/videojs/mux.js/pull/330.

### CODE REVIEW

**Comment by gkatsev** [comment]
Thanks @JulianDomingo. I made a PR that fixes the test: https://github.com/JulianDomingo/mux.js/pull/1

File: lib/tools/caption-packet-parser.js [hunk 1]
```diff
@@ -65,7 +65,18 @@ var parseSei = function(bytes) {
...
+      var userIdentifier = String.fromCharCode(
+        result.payload[3],
+        result.payload[4],
+        result.payload[5],
+        result.payload[6]);
+
+      if (userIdentifier === 'GA94') {
+        break;
+      } else {
+        result.payload = undefined;
```
Comment (gesinger) [inline lib/tools/caption-packet-parser.js:78]:
```suggestion
        result.payload = void 0;
```

**Comment by gkatsev** [comment]
Thanks @JulianDomingo!

**Comment by gkatsev** [comment]
Now published as part of mux.js@5.5.4!

**Pull request merged**